scripts/src/main/resources/scripts/command/aws

```diff
@@ -17,16 +17,28 @@ source "$(dirname "${0}")"/../functions
 # Call aws with specified arguments.
 function doRun() {
   doSetup silent
-  doRunCommand "aws ${*}"
+  if doIsWindows
+  then
+    doRunCommand "\"/c/Program Files/Amazon/AWSCLIV2/aws\" ${*}"
+  else
+    doRunCommand "aws ${*}"
+  fi
 }
 
 
 # OS independent check if aws is installed.
 function doIsAwsInstalled() {
   if command -v aws &> /dev/null
-  then 
+  then
     return
   else
+    if doIsWindows
+    then
+      if [ -f "/c/Program Files/Amazon/AWSCLIV2/aws" ]
```

Comment:
Please use `%ProgramFiles%` and not an exact path.

Reply:
The created version runs without errors at any platform. I'm not able to redesign it with ${PROGRAMFILES}. Please accept this version for now, and please, create a new issue for this point.

Reply:
I think now it works

Reply:
@tobka777 can you approve as this is now perfectly resolved?